internal/cmd/schema-registry/command_cluster_delete.go

```diff
@@ -41,24 +40,26 @@ func (c *command) newClusterDeleteCommand() *cobra.Command {
 }
 
 func (c *command) clusterDelete(cmd *cobra.Command, _ []string) error {
-	ctx := context.Background()
-
 	environmentId, err := c.EnvironmentId()
 	if err != nil {
 		return err
 	}
 
-	cluster, err := c.Context.FetchSchemaRegistryByEnvironmentId(ctx, environmentId)
+	clusterList, _, err := c.V2Client.GetSchemaRegistryClusterByEnvironment(environmentId)
 	if err != nil {
 		return err
 	}
+	if len(clusterList.Data) == 0 {
```

Comment:
Is there a better way to check this? Is there an `enabled` field, or should the backend return an error if it's disabled?